typegate/src/runtimes/wasmedge.ts

```diff
@@ -40,7 +40,8 @@ export class WasmEdgeRuntime extends Runtime {
         await native.wasmedge_wasi(
           {
             func: func as string,
-            wasm: wasm as string,
+            wasm:
+              `tmp/metatype_artifacts/${tg_name as string}/files/${wasm as string}.${artifact_hash as string}`,
```

Comment:
thought: noticed you were reading from `config.tmp_dir` down below. Should it not be the same here?

Reply:
Good eye! didn't make changes after I changed where the path resolves. I will correct it.

Reply:
Done.